Dockerfile

```diff
@@ -70,12 +38,13 @@ RUN git clone --depth 1 -b 2.4.1 https://github.com/modelon-community/fmi-librar
   && mkdir fmi_build && cd fmi_build \
   && mkdir fmi_library \
   && cmake -DCMAKE_INSTALL_PREFIX=/build/fmi-libary/fmi_library .. \
-  && make -j4 \
+  && make -j$(nproc) \
   && make install
 
-RUN git clone --depth 1 -b PyFMI-2.13.1 https://github.com/modelon-community/PyFMI.git \
+RUN git clone --depth 1 -b PyFMI-2.14.0 https://github.com/modelon-community/PyFMI.git \
   && cd PyFMI \
-  && python3 setup.py bdist_wheel --fmil-home=/build/fmi-libary/fmi_library --with-openmp\
+  && python3 setup.py build --fmil-home=/build/fmi-libary/fmi_library -j $(nproc)\
+  && python3 setup.py bdist_wheel --fmil-home=/build/fmi-libary/fmi_library\
```

Comment:
look into if separate `build` and `bdist_wheel` steps are needed.

Reply:
This is done because `bdist_wheel` is single threaded. We can pass `-j` to `build` which allows the task to take ~1/2 the time (168s ->78s) in my testing.